lib/reducers/sort.js

```diff
@@ -21,7 +22,7 @@ const sortReducer = (state = INITIAL_STATE, action) => {
 const getSortedData = ({ data, schema, sort: { ascending, column } }) => {
   if (column) {
     return orderBy(data, item =>
-      result({ ...item, ...schema[column] }, 'accessor', item[column])
+      getValue({ row: item, schema: schema[column], key: column })
```

Comment:
I'd call the iterator argument `row` instead of `item` so the object definition cleans up a bit.

Reply:
I'd also pull the sort direction out into a `const` so the `orderBy` call doesn't need to be split over three lines.

Reply:
i.e.

```
const dir = ascending ? 'asc' : 'desc';
return orderBy(data, item => getValue({ row, schema: schema[key], key }), dir);
```

Reply:
(I want to rename `column` to `key` to clean it even further, but I'm not sure if that's possible, because I'm guessing the `column` comes from the state structure.

Reply:
true that: 
```js
state: {
  ...state,
  sort: {
    column: 'url',
    ascending: false
  }
}
```

can change it to `sort: { key, ascending }` ?